integration_tests/pkg/executor/kubernetes_test.go

```diff
@@ -35,9 +35,11 @@ const (
 func TestKubernetesExecutor(t *testing.T) {
 	// Readable, simple, easy to debug, reproducible and reliable testing environment.
 	log.SetLevel(log.ErrorLevel)
+	// Using timestamps with nanoseconds makes debugging easier.
+	log.SetFormatter(&log.TextFormatter{FullTimestamp: true, DisableTimestamp: false, TimestampFormat: time.RFC3339Nano})
```

Comment:
@skonefal - I do. It was really useful when I was debugging issues with `Execute()`.